abandoned_battlefield/map.conf

```
@@ -0,0 +1,26 @@
+initial_stuff = default:cobble 40,shooter:pistol,default:pick_steel,default:torch 8
+author = HobbitPower
+name = Abandoned Battlefield
+h = 140
+rotation = z
+team.1.pos = (75,1,97)
+r = 115
+team.1 = red
+team.1.color = red
+team.2.pos = (-75,1,-97)
+team.2 = blue
+team.2.color = blue
+license = CC BY-SA 4.0
+hint = Bunkers/shelters and battle tanks can be useful.
```

```suggestion
hint = Find ores hidden on a secret place!
```
I would suggest this hint, modify it if you want. Hint for chest zones isn't needed IMO.

---

```suggestion
hint = Find hidden ores on a secret place!
```